src/components/Navbar/style.css

```diff
@@ -0,0 +1,57 @@
+.App-logo{
```

Comment:
we re gonna learn how to give classNames and how to edit them, but it will be for following project, remember to me, if i forgot